src/TableViewRow.cs

```diff
@@ -9,12 +9,17 @@
 using System.Collections.Specialized;
 using System.Linq;
 using WinUI.TableView.Helpers;
+using WinRT;
 
 namespace WinUI.TableView;
 
 /// <summary>
 /// Represents a row in a TableView.
 /// </summary>
+
+#if WINDOWS
+[WinRT.GeneratedBindableCustomPropertyAttribute]
```

Comment:
since you’re using the fully qualified attribute, the `using WinRT;` at line 12 can be removed. Also, `[WinRT.GeneratedBindableCustomPropertyAttribute]` can be shortened to `[WinRT.GeneratedBindableCustomProperty]` as per IDE0001 rule.